docs/approvals/service-now/index.md

```diff
@@ -42,10 +42,13 @@ The Octopus Deploy / ServiceNow integration requires security configuration in y
 
 Follow the [ServiceNow OAuth documentation](https://docs.servicenow.com/bundle/sandiego-platform-administration/page/administer/security/task/t_SettingUpOAuth.html) to configure an OAuth endpoint for Octopus to use for authentication. Take note of the OAuth client id and client secret from the configuration.
 
-Next, the integration will require a user account on ServiceNow. The recommendation is to ser account specifically for Octopus.
-To create a new ServiceNow user, follow the [ServiceNow user documentation](https://docs.servicenow.com/en-US/bundle/sandiego-platform-administration/page/administer/users-and-groups/task/t_CreateAUser.html). 
```

Comment:
Is there a specific reason to drop this line?

Reply:
Two reasons... (I am completely happy to be wrong mind you)
1. Customers response in https://octopusdeploy.slack.com/files/U018V0KK13J/F03MAQ5LAG0/image2.png
2. The link (like all ServiceNow docs) references a specific version of ServiceNow, What happens when the next version releases, what happens sandiego is deprecated, are we going to remember to update this link.... 🤷‍♂️

Reply:
I had the same thought on the link. I actually added the link based on feedback in the previous PR, but I can agree with your reason